.pre-commit-config.yaml.jinja

```diff
@@ -104,40 +115,45 @@ repos:
       - id: mixed-line-ending
         args: ["--fix=lf"]
   - repo: https://github.com/pycqa/flake8
-    rev: 3.8.3
+    rev: 6.1.0
     hooks:
       - id: flake8
-        name: flake8 except __init__.py
-        exclude: /__init__\.py$
-        additional_dependencies: ["flake8-bugbear==20.1.4", "importlib-metadata<5.0.0"]
+        name: flake8 except __init__.py, __manifest__.py
+        exclude: /__(?:init|manifest)__\.py$
+        additional_dependencies: ["flake8-bugbear==23.7.10", "importlib-metadata<=6.8.0"]
       - id: flake8
         name: flake8 only __init__.py
         args: ["--extend-ignore=F401"] # ignore unused imports in __init__.py
         files: /__init__\.py$
-        additional_dependencies: ["flake8-bugbear==20.1.4", "importlib-metadata<5.0.0"]
+        additional_dependencies: ["flake8-bugbear==23.7.10", "importlib-metadata<=6.8.0"]
+      - id: flake8
+        name: flake8 only __manifest__.py
+        args: ["--extend-ignore=B018"] # ignore found useless expression in __manifest__.py
+        files: /__manifest__\.py$
+        additional_dependencies: ["flake8-bugbear==23.7.10", "importlib-metadata<=6.8.0"]
   - repo: https://github.com/pycqa/pylint
-    rev: v2.11.1
+    rev: {{ proj_rev.pylint }}
     hooks:
       - id: pylint
         name: pylint with optional checks
         args:
-          - --valid_odoo_versions={{ "%.1f"|format(odoo_version) }}
           - --rcfile=.pylintrc
           - --exit-zero
         verbose: true
         additional_dependencies:
-          - isort==4.3.21
-          - pylint-odoo==5.0.5
+          - isort=={{ proj_rev.isort }}
+          - pylint-odoo=={{ proj_rev.pylint_odoo }}
       - id: pylint
         name: pylint with mandatory checks
         args:
-          - --valid_odoo_versions={{ "%.1f"|format(odoo_version) }}
           - --rcfile=.pylintrc-mandatory
+          - odoo/auto/addons
+        verbose: true
         additional_dependencies:
-          - isort==4.3.21
-          - pylint-odoo==5.0.5
+          - isort=={{ proj_rev.isort }}
+          - pylint-odoo=={{ proj_rev.pylint_odoo }}
```

Comment:
what do you think about following OCA and using the pylint_odoo hook directly?

I've had trouble in the past with trying to update this hook and ending up with incompatible versions of pylint and pylint_odoo. Would also simplify the file; the isort dependency is not really necessary since we're running the isort hook separately too.

I understand if it's out of scope though.

Reply:
done!